app/components/Search/ResourcesRow.jsx

```diff
@@ -35,6 +35,55 @@ class ResourcesRow extends Component {
     });
   }
 
+  getOpenTime(schedule) {
+    let date = new Date();
+    let day = date.getDay();
+    let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+    let hour = date.getHours();
+    let closingOpeningTimes = {};
+    let currDay = days[day];
+    let result = { open: false };
+
+    schedule.forEach(item => {
+      closingOpeningTimes[item.day.replace(/,/g, '')] = {
+        open: item.opens_at,
+        close: item.closes_at
+      };
+    });
+
+    if(closingOpeningTimes[currDay] && hour < closingOpeningTimes[currDay].close) {
+      return {
+        open:true, 
+        time: timeToString(closingOpeningTimes[currDay].close)
+      };
+    } else {
+      let remainingDays = days.splice(day+1);
+      remainingDays.some(d => {
+        if(closingOpeningTimes[d]) {
+          result = {
+            open: false,
+            time: `${timeToString(closingOpeningTimes[d].open)} ${d}`
+          };
+          return true;
+        }
+      });
+
+      if(result.time) return result;
+
+      days.some(d => {
```

Comment:
not sure how this is different from the block above (other than splicing out today). If the resource is closed (so return not yet reached) why not just jump to here and skip the else above?